ultimarc/schemas/ultrastik-config.schema

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "required": [
+    "schemaVersion",
+    "resourceType",
+    "deviceClass",
+    "mapSize",
+    "restrictor",
+    "flash",
+    "borders",
+    "map"
+  ],
+  "properties": {
+    "schemaVersion": {
+      "$id": "#root/schemaVersion",
+      "title": "Schemaversion",
+      "type": "integer",
+      "const": 2.0
+    },
+    "resourceType": {
+      "$id": "#root/resourceType",
+      "title": "Resourcetype",
+      "type": "string",
+      "const": "ultrastik-config"
+    },
+    "deviceClass": {
+      "$id": "#root/deviceClass",
+      "title": "Deviceclass",
+      "type": "string",
+      "const": "ultrastik"
+    },
+    "mapSize": {
+      "$id": "#root/mapSize",
+      "type": "number",
+      "const": 9
+    },
+    "restrictor": {
+      "$id": "#root/restrictor",
+      "type": "boolean"
+    },
+    "flash": {
+      "$id": "#root/flash",
+      "type": "boolean"
+    },
+    "keepAnalog": {
+      "$id": "#root/keepAnalog",
+      "type": "boolean"
+    },
+    "borders": {
```

Comment:
The borders can be changed, but I don't think the device works if the borders are anything but these values like map size.  Thinking the same thing here to pull it out of the config, thoughts?

Reply:
I'm not really sure the values can be changed or not, so I'm Ok leaving it in the config.